test/app/components/slim_renders_many_component.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class SlimRendersManyComponent < ViewComponent::Base
+  include ViewComponent::SlotableV2
+
+  renders_many :components, SlimComponent
```

Comment:
```suggestion
  renders_many :slim_components, SlimComponent
```

Thoughts on this naming? It might help make it more explicit that we're rendering slim components as slots.